src/stories/ReactRestPlayground.stories.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import ReactRestPlayground from '../index'
+import '../styles/main.css'
+
+export default {
+  title: 'React Rest Playground',
```

Comment:
change the title `GET Method`

Add for all the types, POST, PATCH, DELETE, etc

Reply:
Have created section `HTTP Methods/` to add `GET, POST, PUT PATCH, DELETE` methods